fact-ebpf/process.h

```diff
@@ -108,7 +109,7 @@ __always_inline static int64_t process_fill(process_t* p) {
 
   unsigned long arg_start = BPF_CORE_READ(task, mm, arg_start);
   unsigned long arg_end = BPF_CORE_READ(task, mm, arg_end);
-  unsigned int len = arg_end - arg_start;
+  unsigned int len = (arg_end - arg_start) & 0xFFFF;
```

Comment:
This highly redundant, but clang 19 was having none of it, so the check needs to stay there.